pmd-core/src/main/java/net/sourceforge/pmd/PMD.java

```diff
@@ -213,10 +213,13 @@ public SourceCodeProcessor getSourceCodeProcessor() {
     /**
      * This method is the main entry point for command line usage.
      *
-     * @param configuration
-     *            the configure to use
+     * @param configuration the configure to use
```

Comment:
Shouldn't it be `the configuration to use`? :)

Reply:
I'll change it, thanks.